### PR TITLE
Update dma-overview.md

### PR DESCRIPTION
**Pull request by siyangMicrosoft**
update dma overview for the SSIS assessment(released in DMA 4.5)

### CODE REVIEW

**Comment by PRMerger14** [comment]
@siyangMicrosoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@siyangMicrosoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/dma/dma-whatsnew.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "What's new in Data Migration Assistant (SQL Server) | Microsoft Docs"
 ms.custom: ""
-ms.date: "10/20/2018"
+ms.date: "07/15/2019"
```
Comment (siyangMicrosoft) [inline docs/dma/dma-whatsnew.md]: close this one.

**Comment by HJToland3** [comment]
#sign-off

**Comment by PRMerger7** [comment]
@siyangMicrosoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger8** [comment]
@siyangMicrosoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/dma/dma-whatsnew.md (hunk 1 above)
Comment (siyangMicrosoft) [reply docs/dma/dma-whatsnew.md]: has merge from Microsoft to my local branch

**Comment by PRMerger18** [comment]
@siyangMicrosoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@siyangMicrosoft @HJToland3 - new articles and large changes to existing articles cannot be made through the public repo. The commits in this PR will need to be re-targeted to the private repo (sql-docs-pr) so that we can take advantage of the staging, proofing, and validation tools there.

We'll close this PR.  
Thanks.

@siyangMicrosoft - here's some information about working in the private repos:
> Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.  
> 
> After you review staging and fix all validation issues, you can tag an author for review or add the sign-off comment to let PR reviewers know that the updates are ready to be merged.  
> 
> Read about [authors working in the public repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master) for more information.  
> 
> If you need help getting set up to work in the private repo, see [How to get to the private repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master#how-to-get-to-the-private-repo). You can also take a look at the [GitHub account setup page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-github?branch=master) for additional details on making sure you're set up.

**Comment by Jak-MS** [comment]
As a follow-up: if you need assistance, we can help you move this to the private repo, but you'll still need access to the private repo in order to see the changes.

**Comment by siyangMicrosoft** [comment]
Hi Jak,

How to get access to the sql-docs-pr?

Thanks,
Siyang

**Comment by Jak-MS** [comment]
Hi @siyangMicrosoft - all the info you need is in my comment here: 

> If you need help getting set up to work in the private repo, see [How to get to the private repo](https://review.docs.microsoft.com/en-us/help/contribute/contribute-authors-in-public-repo?branch=master#how-to-get-to-the-private-repo). You can also take a look at the [GitHub account setup page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-get-started-setup-github?branch=master) for additional details on making sure you're set up.

**Pull request closed without merging**